### PR TITLE
Clarify Go examples.

### PR DESCRIPTION
**Pull request by oliverchang**
`affected.package.name` for Go entries should point to the module, not
to the package import path.

Point to the Go vulnerability database documentation instead of inlining the examples in the spec.

### CODE REVIEW

File: docs/schema.md [hunk 1]
```diff
@@ -816,36 +817,45 @@ applications.  Here is an entry for a recent Go tool vulnerability:
...
-            "severity": "HIGH"
+  "id": "GO-2021-0068",
+  "published": "2021-04-14T20:04:52Z",
+  "modified": "2022-05-23T18:04:20Z",
+  "aliases": [ "CVE-2021-3115" ],
+  "details": "The go command may execute arbitrary code at build time when using cgo on Windows.\nThis can be triggered by running go get on a malicious module, or any other time\nthe code is built.\n",
+  "affected": [
+    {
+      "package": {
+        "name": "std",
```
Comment (rsc) [inline docs/schema.md:828]: The Go security team is still deciding on what this name will be.

**Review by rsc — COMMENTED** [review]
@julieqiu should be able to correct and or confirm the main Go repository example.

File: docs/schema.md (hunk 1 above)
Comment (oliverchang) [reply docs/schema.md]: I've updated everything here to point to Go documentation instead. PTAL.

Comment (oliverchang) [reply docs/schema.md]: @julieqiu fyi

Comment (oliverchang) [reply docs/schema.md]: @rsc friendly ping for review, thanks!

File: docs/schema.md [hunk 2]
```diff
@@ -765,89 +765,9 @@ contains a JSON object with unspecified fields.
...
-                ]
-            }
-        ],
-        "ecosystem_specific": {
-            "severity": "HIGH"
-        }
-    } ]
-}
-```
+The [Go vulnerability database](https://go.dev/security/vulndb/) uses this format.
```
Comment (julieqiu) [inline docs/schema.md:768]: Just a heads up that we changed the canonical URLs for:

https://go.dev/security/vulndb -> https://go.dev/security/vuln.
https://go.dev/security/vulndb/api -> https://go.dev/security/vuln/database

The old links will still redirect.

Comment (oliverchang) [reply docs/schema.md]: Thanks, fixed!

**Review by rsc — COMMENTED** [review]
Perfect, thanks.

**Review by rsc — APPROVED** [review]

**Pull request merged**